coral-dragaction/src/scripts/DragAction.js

```diff
@@ -244,11 +244,13 @@ class DragAction {
 
     this._drag = this._drag.bind(this);
     this._dragEnd = this._dragEnd.bind(this);
+    this._dragOnKeyEnd = this._dragOnKeyEnd.bind(this);
 
     events.on(`touchmove.DragAction${this._id}`, this._drag);
     events.on(`mousemove.DragAction${this._id}`, this._drag);
     events.on(`touchend.DragAction${this._id}`, this._dragEnd);
     events.on(`mouseup.DragAction${this._id}`, this._dragEnd);
+    events.on(`focusout.DragAction${this._id}`, this._dragOnKeyEnd);
```

Comment:
Move this `focusout` event handler to only after the user has started a keyboard drag and drop operation.

```suggestion
```